docs/source/en/using-diffusers/weighted_prompts.md

```diff
@@ -10,10 +10,207 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
+
+New diffusion models do a pretty good job of generating high-quality images from a basic prompt, but it is still important to create a well-written prompt to get the best results. Here are a few tips for writing a good prompt:
+
+1. What is the image *medium*? Is it a photo, a painting, a 3D illustration, or something else?
+2. What is the image *subject*? Is it a person, animal, object, or scene?
+3. What *details* would you like to see in the image? This is where you can get really creative and have a lot of fun experimenting with different words to bring your image to life. For example, what is the lighting like? What is the vibe and aesthetic? What kind of art or illustration style are you looking for? The more specific and precise words you use, the better the model will understand what you want to generate.
+
+<div class="flex gap-4">
+  <div>
+    <img class="rounded-xl" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/diffusers/plain-prompt.png"/>
+    <figcaption class="mt-2 text-center text-sm text-gray-500">"A photo of a banana-shaped couch in a living room"</figcaption>
+  </div>
+  <div>
+    <img class="rounded-xl" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/diffusers/detail-prompt.png"/>
+    <figcaption class="mt-2 text-center text-sm text-gray-500">"A vibrant yellow banana-shaped couch sits in a cozy living room, its curve cradling a pile of colorful cushions. on the wooden floor, a patterned rug adds a touch of eclectic charm, and a potted plant sits in the corner, reaching towards the sunlight filtering through the windows"</figcaption>
+  </div>
+</div>
+
+## Prompt enhancing
+
+Prompt enhancing is a technique for improving prompt quality by using a combination of:
+
+- [*Offset noise*](https://www.crosslabs.org//blog/diffusion-with-offset-noise) improves the contrast in bright and dark images and creates better lighting overall. This is available as a [LoRA](https://hf.co/stabilityai/stable-diffusion-xl-base-1.0/blob/main/sd_xl_offset_example-lora_1.0.safetensors) from [stabilityai/stable-diffusion-xl-base-1.0](https://hf.co/stabilityai/stable-diffusion-xl-base-1.0).
+- GPT2 enhances the original prompt with a list of words you want. This means curating a list of specific keywords and forcing the model to use those words in the original prompt. As a result, you can use a very basic prompt and still get very high-quality results without spending too much effort. For example, your prompt can be "a cat" and GPT2 can improve it to "cinematic photo of cat, 35mm photograph, film, professional, 4k, highly detailed, breathtaking, hyperrealistic, stunning, ultra, dynamic".
```

Comment:
to break down techniques cc @asomoza here if I'm wrong
1. provide a list of words to the model and motivate it to generate a prompt using these words
2. apply some post-processing 
3. use "prompt template"
4. use the noise offset model (I don't think it counts as a prompt technique though)

Reply:
I'll gladly do it but the huggingface posts are very limited with the markdown and the images , so it practically has to be all text with the images as attachments at the end.

Reply:
Totally! I like the option of having it as a blog post where @asomoza can explain all the design decisions and details. 

This narrative-driven content style (what the problem is, how you solved it, etc.) is more appropriate as a Hugging Face blog post or Hub post imo. The "why" of prompt enhancing and walking through the process of it works better in blog posts where you can add additional context or opinion about the feature (we can link to it from the doc), and it can also help inspire users without getting in the way of users who are looking for something specific in the docs to get started.